README.md

```diff
@@ -113,25 +113,30 @@ exemple of default template in JS :
 
 ```javascript
 // load the Kuzzle SDK module
-const Kuzzle = require('kuzzle-sdk')
+const Kuzzle = require('kuzzle-sdk').Kuzzle;
```

Comment:
nitpicking: official node js doc prefers  `const { Kuzzle } = require('kuzzle-sdk');` (cf for instance https://nodejs.org/docs/latest-v9.x/api/crypto.html#crypto_certificate_exportchallenge_spkac)